pkg/csi/service/wcp/controller_helper.go

```diff
@@ -440,3 +443,88 @@ func getOverlappingNodes(accessibleNodes []string, topologyRequirement *csi.Topo
 	}
 	return overlappingNodes, nil
 }
+
+// canFakeAttach checks if the volume is eligible to be fake attached and returns a bool value
+func canFakeAttach(ctx context.Context, volumeID string, volumeManager cnsvolume.Manager) bool {
```

Comment:
This method assumes "pvc". See my other comments on how this could be generalized.